<!DOCTYPE html>

<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <title>MsDBP</title>
    <style type="text/css">
        body{
           background-repeat:repeat-x;
           border-collapse:collapse;
           color:#000;
           font-family:"Helvetica Neue", "Helvetica", "Arial", sans-serif;
           font-size:15px;
           line-height:1.5em;
           margin:3% 10% 5% 10%;
              /*max-width: 960px;*/
           padding:0px;
        }
        a{
           font-weight:bold;
        }
    </style>
    <meta name="Keywords" content="DNA binding protein, deep learning, multi-scale feature">
</head>


<body bgcolor="#FFE4C4" style="font-family:Arial, Helvetica, sans-serif">
    <span style="font-size:30px;color:green;font-weight:bold;font-style:italic;">MsDBP</span><br><br>
 
    <h2 style="font-size:18px">Abstractï¼š</h2>
    <p  style="text-align:justify; text-justify:inter-ideograph;">
        DNA-binding proteins are crucial to alternative splicing, methylation and the structural composition of the DNA. The existing experimental methods for identifying DNA-binding proteins are expensive and time-consuming and thus it is necessary to develop a fast and accurate computational method to address the problem. In this paper, we report a novel predictor MsDBP, a DNA-binding protein prediction method that combines the multi-scale sequence feature into a deep neural network. First of all, instead of developing a narrow-application structured-based method, we are committed to a sequenced-based predictor. Secondly, instead of characterizing the whole protein directly, we divide the protein into subsequences with different lengths and then encode them into a vector based on composition information. In this way, multi-scale sequence feature can be obtained. Finally, a branch of dense layers is applied for learning diverse multi-level abstract features to discriminate DNA-binding proteins. When MsDBP is tested on the independent dataset PDB2272, it achieves an overall accuracy of 66.99% with the F1 of 68.49%. In addition, we also perform extensive experiments to compare the proposed method with other existing methods. The results indicate that MsDBP would be a useful tool for the identification of DNA-binding proteins.
        <br>
    </p>

        <TR><td>

            <h2 style="font-size:18px">Following steps should be followed while using MsDBP server:</h2>

            <ul>

                <li><p align="justify"><font color="#0000FF">Step one:</font> Clicking the link 
                  <a href="{% url 'MsDBP' %}">here</a> to open the web-server and you will see the home page.</p> </li>  

                <li><p align="justify"><font color="#0000FF">Step two:</font> Directly paste protein sequences into text-box provided. Sequences must be in <b>FASTA</b> format. All non standard amino acids (such as "X" and "Z") will be ignored from the sequences. </p></li>

                <li><p align="justify"><font color="#0000FF">Step three:</font> click the <b>Run</b> button to submit the protein sequences to the server, then you will get the prediction results . </p></li>
            </ul>
        </td></TR>
       
</body></html>